test/functional/TestUtilitiesJ9/src/org/openj9/test/attachAPI/AttachApiTest.java

```diff
@@ -224,8 +224,10 @@ protected List<String> runCommand(List<String> args) throws IOException {
 
 	protected List<String> runCommandAndLogOutput(List<String> commandLineOptions) throws IOException {
 		// this api is used by jcmd and jps
-		commandLineOptions.add("-J-Dcom.ibm.tools.attach.logging=yes");
-		List<String> jpsOutput = runCommand(commandLineOptions);
+		List<String> commandLineArgs = new ArrayList<>();
+		commandLineArgs.addAll(commandLineOptions);
+		commandLineArgs.add("-J-Dcom.ibm.tools.attach.logging=yes");
```

Comment:
Suggest reversing these which would allow a test to disable logging.

Reply:
Reversed the order.